datarepo-clienttests/src/main/java/utils/FileUtils.java

```diff
@@ -0,0 +1,50 @@
+package utils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.net.URL;
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public final class FileUtils {
+
+  private FileUtils() {}
+
+  private static SecureRandom randomGenerator = new SecureRandom();
+
+  public static String randomizeName(String baseName) {
+    long suffix = randomGenerator.nextLong();
+    return baseName + (suffix <= 0 ? -suffix : suffix);
```

Comment:
```suggestion
    return baseName + Long.toUnsignedString(suffix);
```

Reply:
Since that method doesn't preserve the sign, I think it would be effectively halving the number of random suffixes I could generate.